tableauserverclient/server/endpoint/workbooks_endpoint.py

```diff
@@ -87,11 +87,16 @@ def update(self, workbook_item):
         return updated_workbook._parse_common_tags(server_response.content, self.parent_srv.namespace)
 
     # Update workbook_connection
-    def update_conn(self, workbook_item, connection_item):
+    @api(version="2.3")
+    def update_connection(self, workbook_item, connection_item):
```

Comment:
Should probably set `update_conn` to `update_connection` for back compact; and throw a warning?

Reply:
Probably... I'll make the change.